code/game/machinery/suit_storage_unit.dm

```diff
@@ -258,16 +278,16 @@
 */
 /obj/machinery/suit_storage_unit/proc/cook()
 	var/mob/living/mob_occupant = occupant
+	var/burn_damage = laser_strength
+	if(uv_super || (obj_flags & EMAGGED))
```

Comment:
you could use a ternary here `var/burn_damage = uv_super || (obj_flags & EMAGGED) ? laser_strength_hacked : laser_strength`